.github/workflows/cpp_lint.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Format and Lint C++ Code
-      uses: DoozyX/clang-format-lint-action@v0.11
+      uses: DoozyX/clang-format-lint-action@v0.18.1
```

Comment:
Could you please create a separate PR for updating `.github/workflows/cpp_lint.yml`? Why is updating the versions necessary for this PR?

Reply:
The CPP linter was erroring out with
``` 
Traceback (most recent call last):
  File "/run-clang-format.py", line 25, in <module>
    from distutils.util import strtobool
ModuleNotFoundError: No module named 'distutils'
```
You can see that other outstanding PRs are also failing with the same error. I changed the version in order to fix the error.